src/view/view-layout.js

```diff
@@ -0,0 +1,75 @@
+import _ from 'underscore';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import Sink from './sink';
+import sinkLayoutGen from './sink-layout-gen';
+
+class View extends Component {
```

Comment:
The whole point of it is, that I basically need `ViewLayout` to get completely thrown away whenever the job_id changes. The `componentCharts` array needs to get reset and componentCharts need to get properly disposed. Doing it other way would require some propsDidChange bullshit.